### DISCUSSION TITLE
How to restart Octolapse? 'Timelapse cancelled' won't go away

### DISCUSSION THREAD

**Discussion by titaniumschmapple** [FormerLurker/Octolapse#797] [Q&A]
Hi,

How do I restart Octolapse? 'Timelapse cancelled' won't go away after I cancelled it previously (only because the previous print had finished and Octolapse hadn't created a movie from it).

Thanks!

**Comment by FormerLurker** [comment]
Hmm..  I'm not sure exactly, but I'm guessing there would be an error in the javascript console (F12 to open, select console tab) that could shed some light on this.  I'm guessing ctrl+f5 would clear the error (hard refresh) as a workaround.
